ui/appui-layout-react/src/appui-layout-react/base/NineZoneState.ts

```diff
@@ -1022,10 +1023,12 @@ export function createPopoutWidgetState(id: PopoutWidgetState["id"], args?: Part
 
 /** @internal */
 export function createTabState(id: TabState["id"], args?: Partial<TabState>): TabState {
+  const hideWithUiWhenFloating = args?.hideWithUiWhenFloating ? args.hideWithUiWhenFloating : false;
```

Comment:
Since `hideWithUiWhenFloating` is optional, no need to initialize to `false`, since we handle `false` and `undefined` cases identically?

Reply:
I'm sure you're right, but leaving things uninitialize/set to undefined bugs me.

Reply:
Done!